### DISCUSSION TITLE
anyone could write a tutorial for this stuff would be appreicated

### DISCUSSION THREAD

**Discussion by by6et** [ghokun/appletv3-iptv#4] [General]
As a newbie to apple tv3 and linux , seems this should work on a linux server or something like that? not sure.  Please, if someone
could write a detail for this would be greatly appreciated.

**Comment by ghokun** [comment]
Hi! Thanks for the interest for the project.

You need three things

- A DNS server (Your router may have this feature)
- A device to run appletv3-iptv application (Rasberry pis are great because it needs to run 7/24 or atleast when you use your apple tv)
- Apple TV 3

If you have more questions don't hesitate to ask.
